libs/ui/src/tokens/figma/dark/variables.css

```
@@ -1004,17 +1034,21 @@
   --radius-textarea-lg: 0.75rem;
   --radius-textarea-md: 0.5rem;
   --radius-textarea-sm: 0.25rem;
-  --radius-toast: 0.25rem;
+  --radius-toast-root: 0.25rem;
   --radius-tooltip: 0.5rem;
   --radius-tree-view: 0rem;
+  --radius-xs: 0.13rem;
   --ring-offset: 0.13rem;
   --ring-offset: 0.13rem;
```

Stylelint (17.12.0)

[error] Duplicate custom property "--ring-offset" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)